common/environment/dalienv.hpp

```diff
@@ -53,4 +53,10 @@ extern ENVIRONMENT_API void envInstallNASHooks(IPropertyTree *nasPTree, SocketEn
 extern ENVIRONMENT_API IPropertyTree *envGetInstallNASHooks(SocketEndpoint *myEp=NULL);
 extern ENVIRONMENT_API IPropertyTree *envGetInstallNASHooks(IPropertyTree *nasPTree, SocketEndpoint *myEp=NULL);
 
+#else
+
+inline bool canSpawnChildProcess(const IpAddress & ip) { return true; }
```

Comment:
I prefer as little conditionally compiled code as possible - it makes the code easier to read, and ensures it compiles with all the different options.  So I prefer a dummy implementation in containerized to having #ifs around the call.
This PR in general is slightly different because it is removing a dependency on a dll, but I would hope some of the conditional compiles can be replaced by general functions instead (that call either bare metal or containerized implementations.)